### ISSUE TITLE
Datadog metrics forwarding roadmap

### ISSUE THREAD

**Issue by BrynCooke** [apollographql/router#3958]
Currently we have two types of metrics:
* Protobuf based
* Otel

We have a feature that allows conversion and forwarding of our protobuf based metrics to datadog, https://www.apollographql.com/docs/graphos/metrics/datadog-integration/#forwarded-metrics

However this does not contain otel metrics, of which there are more and more of over time. This is mostly because adding otel metrics are very easy, and adding protobuf metrics is hard.

We should have a roadmap for what we want to do with this longer term as the current model of two different metrics sources doesn't provide a good user experience.

**Comment by abernix** [comment]
Agree, but closing as this won't get solved here on `router` repo.
